sys/include/net/gnrc/netdev2.h

```diff
@@ -102,6 +102,25 @@ typedef struct gnrc_netdev2 {
      * @brief general information for the MAC protocol
      */
     uint16_t mac_info;
+
+#if ((GNRC_MAC_RX_QUEUE_SIZE != 0) || (GNRC_MAC_DISPATCH_BUFFER_SIZE != 0))
```

Comment:
It is already in the field of [#ifdef MODULE_GNRC_MAC](https://github.com/RIOT-OS/RIOT/pull/6074/files#diff-49911bcb6099f3f968cf91ed2a95ca21R100). :-)